lib/metro-transit.js

```diff
@@ -16,23 +16,21 @@ var MetroTransit = function () {
 MetroTransit.prototype = Object.create(AlexaSkill.prototype);
 MetroTransit.prototype.constructor = MetroTransit;
 
```

Comment:
Let me try and run the debugger on what `MetroTransit.prototype.eventHandlers` is at this point. I bet there is more to this object and we were overwriting it.